mailpoet/assets/js/src/segments/dynamic/templates/index.tsx

```diff
@@ -0,0 +1,195 @@
+  },
+] as const;
+
+const templates = [
+  {
+    id: 1,
+    name: 'Welcome series for new users',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: true,
+    isEngaged: false,
+  },
+  {
+    id: 2,
+    name: 'Welcome series for new users',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: true,
+    isEngaged: false,
+  },
+  {
+    id: 3,
+    name: 'Welcome series for new users',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: true,
+    isEngaged: false,
+  },
+  {
+    id: 4,
+    name: 'Welcome series for new users',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: true,
+    isEngaged: false,
+  },
+  {
+    id: 5,
+    name: 'Welcome series for new users',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: true,
+    isEngaged: false,
+  },
+  {
+    id: 6,
+    name: 'Clickers',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: false,
+    isEngaged: true,
+  },
+  {
+    id: 7,
+    name: 'Recent clickers',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: false,
+    isEngaged: true,
+  },
+  {
+    id: 8,
+    name: 'Non-Openers',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: false,
+    isEngaged: true,
+  },
+  {
+    id: 9,
+    name: 'Recent openers',
+    description:
+      'Subscribers who have not engaged with your store or campaigns in the past 30 days.',
+    isEssential: false,
+    isEngaged: true,
+  },
+] as const;
+
+export function SegmentTemplates(): JSX.Element {
+  return (
+    <div className="mailpoet-templates-container">
+      <HideScreenOptions />
+      <TopBarWithBeamer />
+      <Flex className="mailpoet-templates-heading" direction="row" gap="16px">
+        <FlexBlock>
+          <Button
+            icon={chevronLeft}
+            aria-label={__('Navigate to the segments list page', 'mailpoet')}
+            href="#/"
+            label={__('Segments list', 'mailpoet')}
+          />
+          <h1 className="wp-heading-inline">
+            {__('Start with a pre-built segment', 'mailpoet')}
+          </h1>
+        </FlexBlock>
+
+        <FlexItem>
+          <SearchControl label={__('Search segment templates', 'mailpoet')} />
+        </FlexItem>
+
+        <FlexItem>
+          <Button variant="secondary" href="#/new-segment">
+            {__('Create custom segment', 'mailpoet')}
```

Comment:
The "back" button on the new segment page always goes to the segment listing page, so we might want to update it so a user returns to the templates page if they get there from here.

Reply:
My assumption is that the segments page will work the same way that the automations page work and that the "New segment" button in the segment listing page will always direct users to the templates page. So, what I believe we need to do is, in a subsequent PR, when we launch the templates page, we need to update the back button on the new segment page. Let me know if that is not the case, and I'm missing something.

Reply:
That makes perfect sense, thanks for the explanation!